data_ingestion/03_Ingest_data_with_Redshift_v3.ipynb

```diff
@@ -0,0 +1,1956 @@
+{
```

Comment:
<p>Is IAMFullAccess required for this notebook? If so, we should explicitly say this here or above if you have not already (I didn't see it). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>